lib/doorkeeper/exception_raiser.rb

```diff
@@ -0,0 +1,32 @@
+module Doorkeeper
+  class ExceptionRaiser
+    attr_reader :error
```

Comment:
It looks like this attribute is only used within the class scope, therefore it might be nice to make it private, what do you think?